include/oneapi/dpl/pstl/iterator_impl.h

```diff
@@ -75,6 +76,38 @@ struct __make_references
     }
 };
 
+template <typename _Iter, typename _Void = void>
```

Comment:
Probably good enough to write this like
```suggestion
template <typename _Iter, typename = void>
```